examples/resources/awscc_ec2_flow_log/cloudwatch.tf

```diff
@@ -0,0 +1,48 @@
+resource "awscc_ec2_flow_log" "example" {
+  deliver_logs_permission_arn = awscc_iam_role.example.arn
+  log_destination_type        = "cloud-watch-logs"
+  log_destination             = awscc_logs_log_group.example.arn
+  traffic_type                = "ALL"
+  resource_id                 = "vpc-07ddade55bee92f5f"
```

Comment:
@BondAnthony 
- Can we make the the `resource_id` a variable than the vpc_id present here ?
- Can we add tags for ec2_flow_log resource , as some in the schema has some differences for some resources ?

Reply:
I've added tags to the example and updated the `resource_id` to use a variable.